### ISSUE TITLE
Lint errors

### ISSUE THREAD

**Issue by KenCoder** [garris/BackstopJS#741]
Even though the project asks submitters to lint their code, it's not actually being done. Running lint-node  on the current code changes 14 files with lint fixes and leaves 6 unhandled errors. Also the lint-web is linting some extra garbage, a simple fix to the invocation can fix that. 

I'm happy to submit a PR cleaning up the lint. To keep it clean going forward, I think it should be checked as part of build - perhaps fail the travis build?

**Comment by garris** [comment]
Yes, would be great to fix the garbage and also to add the command to the documentation -- these two issues are probably the main reason that no one is using it.

I would hold off on failing travis since it will end up being unplanned work for me. 

On that note however, I would really love to run the sanity-test and the smoke-test on travis -- that is something that I would really want to know is working when we push to GH.

**Comment by KenCoder** [comment]
Of course, it's up to you, but if I actually clean up all the linted items and add the code to fail in Travis if lint fails, why is it extra (unplanned) work for you?

Maybe I will look at the smoke test for you. Note that the current smoke test fails on my mac - times out on the 'click' test.

**Comment by garris** [comment]
Sure, so the unplanned bit will be me having to fix Travis reports if someone forgets a semicolon somewhere.  It’s not a big deal — but it would be adding complexity for a problem that really isn’t a problem.   On the other hand,  if the feature tests fail I really do want Travis to complain about it 😄

The current master branch should have a fix for the failing feature tests you mentioned.  I was using that to test that error reporting was actually fixed and I forgot to roll the config back before bumping versions.
